### PR TITLE
Added queues status to AgentDesktopPanel when there are no tasks

### PR DESCRIPTION
**Pull request by GPaoloni**
Jira: https://bugs.benetech.org/browse/CHI-215

Added queues status to AgentDesktopPanel when there are no tasks

Screenshot from 2020-09-04 11-54-47

### CODE REVIEW

**Comment by nickhurlburt** [comment]
If you send a chat, then set yourself available, there's some unneeded padding on the right side:

image

Otherwise this is good.

**Comment by nickhurlburt** [comment]
I'll go over these two options in our design meeting tomorrow.

File: plugin-hrm-form/src/utils/setUpComponents.js [hunk 1]
```diff
@@ -60,6 +60,28 @@ export const setUpQueuesStatus = () => {
...
+        smsColor,
+        whatsappColor,
+      }}
+      noActiveTasks
+    />,
+    {
+      sortOrder: -1,
+      align: 'start',
+      if: props => {
+        console.log(props);
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/utils/setUpComponents.js:80]: We may not want to keep this.

**Review by nickhurlburt — APPROVED** [review]
Let's go with this one!

**Comment by GPaoloni** [comment]
Deployed

**Pull request merged**